src/lib/timing.c

```diff
@@ -0,0 +1,207 @@
+/*
+   +----------------------------------------------------------------------+
+   | Xdebug                                                               |
+   +----------------------------------------------------------------------+
+   | Copyright (c) 2002-2020 Derick Rethans                               |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 1.01 of the Xdebug license,   |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available at through the world-wide-web at                           |
+   | https://xdebug.org/license.php                                       |
+   | If you did not receive a copy of the Xdebug license and are unable   |
+   | to obtain it through the world-wide-web, please send a note to       |
+   | derick@xdebug.org so we can mail you a copy immediately.             |
+   +----------------------------------------------------------------------+
+   | Authors: Derick Rethans <derick@xdebug.org>                          |
+   |          Michael Voříšek <mvorisek@mvorisek.cz>                      |
+   +----------------------------------------------------------------------+
+ */
+
+#include "php_xdebug.h"
+#if PHP_WIN32
+	#include "win32/time.h"
+	#include <versionhelpers.h>
+#else
```

Comment:
Preprocessor defines aren't normal control structures, so don't format them as they were. The `#` also needs to be in the first column. The other code in Xdebug would have formatted the above as:

```
#if PHP_WIN32
# include "win32/time.h"
# include <versionhelpers.h>
#else 
```